### ISSUE TITLE
What is the number of epoch of the experiment?

### ISSUE THREAD

**Issue by BigChen28** [JiahuiYu/wdsr_ntire2018#15]
Hi, Jiahui.
Could you tell me how many epoches the L1 loss convergence or PSNR reaches the optimal value when you train WSDR_B?

**Comment by JiahuiYu** [comment]
@BigChen28 In all ablation experiments we use same 250 epochs (Table 1, Figure 3 and 4).

Meanwhile, I find longer training epochs help to WDSR (while to EDSR the performance is similar). I guess it is mainly because we use 10x larger learning rate with weight normalization.

**Comment by BigChen28** [comment]
I got it,thanks for your reply.
